content/docs/command-reference/pull.md

```diff
@@ -37,17 +38,17 @@ remote.
 
 With no arguments, just `dvc pull` or `dvc pull --remote <name>`, it downloads
 only the files (or directories) missing from the workspace by searching all
-[DVC-files](/doc/user-guide/dvc-file-format) currently in the
+[`.dvc` files](/doc/user-guide/dvc-file-format) currently in the
```

Comment:
Searching all stages in dvc.yaml or .dvc files currently in ...

Reply:
Please try apply this logic to the rest of this doc as well as for push @utkarshsingh99 and lmk when to review again. I can fix details then. Thanks

Reply:
Should I review again? I know you commented so but that was before this comment so I'm not sure. Feel free to use the PR "request review" to signal this:

![image](https://user-images.githubusercontent.com/1477535/84092243-687a1b00-a9bc-11ea-850f-da16d6fcd57c.png)